src/WattleScript.Tests/EndToEnd/CLikeTestRunner.cs

```diff
@@ -24,7 +24,7 @@ public async Task RunThrowErros(string path)
         await RunCore(path);
     }
 
-    [Test, TestCaseSource(nameof(GetTestCases))]
+    //[Test, TestCaseSource(nameof(GetTestCases))]
```

Comment:
Mistake?

Reply:
Intentional, need to change how this works but out of scope of this PR, disabled for now